### ISSUE TITLE
Is it possible to prompt a user to log in from a dapp they have approved?

### ISSUE THREAD

**Issue by realisation** [MetaMask/metamask-extension#6617]
I have a dapp and I want it to be able to know when the account is locked and be able to have metamask throw up a prompt for the user to unlock their account. They have already logged in before and allowed the dapp to talk to metamask, but then they hit the log out button or they have quit their browser and are not re-opening it.

I've been reading a few issues where .enable is supposed to open a prompt, but it doesn't do anything for me, possibly because the app is already enabled. 

Is it possible to prompt metamask to ask for unlocking, or do I just need to instruct the user to do so themselves?

**Comment by tmashuang** [comment]
What about ?
```
if (ethereum.selectedAddress !== web3.eth.accounts[0]) {
  promptLoginMetaMask()
}
```

or 

```
if (await ethereum._metamask.isApproved() !== await ethereum._metamask.isUnlocked) {
  promptLoginMetamask()
}
```

**Comment by tmashuang** [comment]
Implementation seems to have been changed/fixed with v6.5.3

**Comment by realisation** [comment]
No... I am asking if we can cause metamask to make a prompt for the login information on its own.

I'm not confused that I can make something in my own app to tell the user to log in -  I'd like metamask to skip the step and just open up for login.

What is this "promptLoginMetamask()"?

**Comment by realisation** [comment]
@tmashuang

**Comment by realisation** [comment]
Yes, it looks like it was fixed in v6.5.3 - it works in my Chrome but my Brave browser has not updated for some reason and is still experiencing the problem.
